test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -440,6 +449,36 @@ TEST_P(OverrideHostTest, ClusterGoneHostStays) {
       GetAffinityCookieHeaderForBackend(DEBUG_LOCATION, 1, kNumEchoRpcs / 3));
 }
 
+TEST_P(OverrideHostTest, DisabledPerPath) {
```

Comment:
This test should be called `EnablePerRoute`.

Reply:
Done